tox.ini

```diff
@@ -0,0 +1,30 @@
+[tox]
+skipsdist = True
+envlist = 
+    py38-django-wagtail{211,213}
```

Comment:
I've kept the envlist constrained for now. I don't see added value in having a massive envlist in the current fase of early development.

I do intend to add different Django versions to the envlist sometime in the future.